.pre-commit-config.yaml

```diff
@@ -1422,4 +1414,12 @@ repos:
         require_serial: true
         pass_filenames: false
         files: ^airflow-core/src/airflow/config_templates/config\.yml$
+      - id: generate-airflowctl-help-images
```

Comment:
Why moving it here? Only CI-image-dependent pre-commits should be here (see the comment above)

Reply:
Answered in below comment, converting this back and try another way